### PR TITLE
feat(common):added new component years button[SEM-09]

### PR DESCRIPTION
**Pull request by 9lnmune9**
Added new component years button and filter for years

Volumens_Button Anios
FilterAnios
FilterAnios2

### CODE REVIEW

**Comment by olmedo90** [comment]
image
It is according to the figma, continue with the same interest.
👍👍👍

File: src/app/volumenes/[dateYear]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import volumenes from "@/data/Volumenes";
+
+const Volume = ({ params }: any) => {
+  // Verificar si params está definido y tiene la propiedad dateYear
+  if (!params || !params.dateYear) {
+    // Mostrar todos los datos de volumenes
+    return (
+      <div className="flex min-h-screen flex-col items-center justify-between p-24">
```
Comment (Ditmar) [inline src/app/volumenes/[dateYear]/page.tsx:8]: please fragment, your components, into small parts. 
you can divide this part of your code and create a component called `ItemVolumes`

File: src/app/volumenes/[dateYear]/page.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  // Si params.dateYear está definido, filtrar y mostrar datos específicos
+  return (
+      <div className="flex min-h-screen flex-col items-center justify-between p-24">
+        {volumenes
+          .filter((volume) => volume.dateYear === params.dateYear)
+          .map((volume) => (
+            <div key={volume.id} className="flex flex-col border-2 border-gray-300 rounded-lg p-4 m-4">
+              <h3 className="text-xl font-bold">
```
Comment (Ditmar) [inline src/app/volumenes/[dateYear]/page.tsx:33]: The same here is important to divide your code an create a reusable components

File: src/app/volumenes/page.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+
+import volumenes from "@/data/Volumenes"
```
Comment (Ditmar) [inline src/app/volumenes/page.tsx:2]: please use simple quotes

File: src/app/volumenes/page.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import Volume from "./[dateYear]/page"
```
Comment (Ditmar) [inline src/app/volumenes/page.tsx:3]: use simple quotes

**Review by Ditmar — CHANGES_REQUESTED** [review]
good work! please review the few comments, and you can get the approve in this pr

**Review by Ditmar — APPROVED** [review]
Approved good job

**Pull request merged**